cli/args/flags.rs

```diff
@@ -535,7 +535,8 @@ static ENV_VARIABLES_HELP: &str = r#"ENVIRONMENT VARIABLES:
     NPM_CONFIG_REGISTRY  URL to use for the npm registry.
     NO_COLOR             Set to disable color
     NO_PROXY             Comma-separated list of hosts which do not use a proxy
-                         (module downloads, fetch)"#;
+                         (module downloads, fetch)
+    V8_FLAGS             Set V8 command line options"#;
```

Comment:
Should this be `DENO_V8_FLAGS`? The non-`DENO_` prefixed env vars are for general stuff that might apply to other processes, but this one seems specific to Deno like the other `DENO_` prefixed names.

Reply:
done